pandas/core/indexes/interval.py

```diff
@@ -977,6 +977,14 @@ def interval_range(
         Whether the intervals are closed on the left-side, right-side, both
         or neither.
 
+        .. deprecated:: 1.5.0
+           Argument `closed` has been deprecated to standardize boundary inputs.
```

Comment:
we need 1 tests to assert that `closed` is showing the proper deprecation message.
we need a test that asserts if *both* are passed there is an error raised

Reply:
ok. Have updated.

Reply:
Have updated. Done